code/modules/projectiles/projectile.dm

```diff
@@ -693,11 +693,13 @@ So if we are on the 32th absolute pixel coordinate we are on tile 1, but if we a
 	return TRUE
 
 /obj/machinery/deployable/mounted/projectile_hit(obj/projectile/proj, cardinal_move, uncrossing)
+	. = ..()
+	var/hit_chance = coverage
+	hit_chance = min(hit_chance , hit_chance + 100 - proj.accuracy)
+	if(!density)
+		return prob(hit_chance)
```

Comment:
doesn't this mean the IFF functionality for not hitting deployables stops working when mounted?
i.e. put this check after the iff if statement